src/uct/ugni/rdma/ugni_rdma_ep.c

```diff
@@ -302,6 +295,14 @@ ucs_status_t uct_ugni_ep_atomic_cswap64(uct_ep_h tl_ep, uint64_t compare, uint64
     return uct_ugni_post_fma(iface, ep, &fma->super, UCS_INPROGRESS);
 }
 
+#if GNI_VERSION < GNI_VERSION_CHECK(0x00, 0x5c, 0x0101)
```

Comment:
@hppritcha is there a better way to detect that a version of ugni library supports Aries interconnect ?